mesh/mesh.hpp

```diff
@@ -1364,8 +1376,23 @@ class GeometricFactors
       DETERMINANTS = 1 << 2,
    };
 
+   GeometricFactors() = default;
+
    GeometricFactors(const Mesh *mesh, const IntegrationRule &ir, int flags);
 
+   GeometricFactors(const GridFunction *nodes, const IntegrationRule &ir,
+                    const int flags);
+
+   //Overrides calcuating geometric factors with mesh nodes and uses user defined nodes
+   void AssembleWithNodes(const GridFunction *nodes, const IntegrationRule &ir,
+                          const int flags);
+
+   void Recompute();
+
+   void SetForRecompute(bool recompute=false) {recompute_factors = recompute; };
+
+   bool RecomputeStatus() {return recompute_factors; };
```

Comment:
Symmetrize spaces.

Reply:
Do you mean change this `` {return recompute_factors; };`` to this `` {return recompute_factors;};`` ?

Reply:
either one, we can find both throughout the code, pick the one you prefer.